sklearn/metrics/classification.py

```diff
@@ -2392,6 +2392,6 @@ def brier_score_loss(y_true, y_prob, sample_weight=None, pos_label=None):
                 np.array_equal(labels, [-1])):
             pos_label = 1
         else:
-            pos_label = y_true.max()
```

Comment:
why?

Reply:
for me it makes more sense to get the maximum from the labels (unique y_true) than from y_true itself, the result would be the same but in some cases, it would be faster

Reply:
fair.